sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -535,3 +538,75 @@ def _to_generated(self) -> GeneratedConfigurationSnapshot:
             retention_period=self.retention_period,
             tags=self.tags,
         )
+
+
+class ConfigurationSettingPropertiesPaged(PageIterator):
```

Comment:
Given it is a `PageIterator`, should we have "PageIterator" in the class name?

Reply:
The naming should be good, I'm following the naming in [Tables](https://github.com/Azure/azure-sdk-for-python/blob/2d44f8dac7e11d187537c825f7745861d795dab4/sdk/tables/azure-data-tables/azure/data/tables/_models.py#L278)